modules/logging-release-notes-5-6-22.adoc

```diff
@@ -0,0 +1,17 @@
+// module included in /logging/logging-5-6-release-notes
+:_mod-docs-content-type: REFERENCE
+[id="logging-release-notes-5-6-22_{context}"]
+= Logging 5.6.22
+This release includes link:https://access.redhat.com/errata/RHBA-2024:5125[OpenShift Logging Bug Fix 5.6.22]
```

Comment:
This link opens a 404.

Reply:
Hi @lahinson! Once the release is live, the link will work. :)